.github/workflows/benchmark_long.yml

```diff
@@ -82,6 +83,25 @@ jobs:
       e2e-test-artifacts-gcs-artifact-dir: ${{ needs.build_e2e_test_artifacts.outputs.e2e-test-artifacts-gcs-artifact-dir }}
       benchmark-tools-gcs-artifact-dir: ${{ needs.build_benchmark_tools.outputs.benchmark-tools-gcs-artifact-dir }}
 
+  process_benchmark_results:
+    needs: [setup, compilation_benchmarks, execution_benchmarks]
+    if: fromJson(needs.setup.outputs.should-run) && needs.setup.outputs.benchmark-presets != ''
+    runs-on:
+      - self-hosted # must come first
+      - runner-group=${{ needs.setup.outputs.runner-group }}
+      - environment=${{ needs.setup.outputs.runner-env }}
+      - cpu
+      - os-family=Linux
+    steps:
+      - name: "Checking out repository"
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+      - name: Trigger DB import
+        continue-on-error: true
```

Comment:
What's the reason we put `continue-on-error` here?

Reply:
As long as the cloud functions are not yet production ready I would prefer not to fail the workflow if the imports fail. This will of course be removed at some point.

Reply:
I added a TODO